### PR TITLE
Add Arengu to form services list documentation

### PR DESCRIPTION
**Pull request by jacobovidal**
Add Arengu to form services list documentation

### CODE REVIEW

File: docs/pages/resources.md [hunk 1]
```diff
@@ -49,6 +49,7 @@ Use a SaaS service as a backend for functionality on your Jekyll site
   - [Snipcart](https://snipcart.com/blog/static-site-e-commerce-part-2-integrating-snipcart-with-jekyll): Add a shopping cart to a Jekyll site
 
 ### Forms
+  - [Arengu] (https://www.arengu.com)
```
Comment (MichaelCurrin) [inline docs/pages/resources.md:52]:
```suggestion
  - [Arengu](https://www.arengu.com)
```

Whitespace is invalid so I removed. Check the markdown view to see how it looks with whitespace

https://github.com/jekyll/jekyll/blob/7356006d7019dbcd038212bd18a2d83d7f7f11cd/docs/pages/resources.md#forms

**Review by MichaelCurrin — CHANGES_REQUESTED** [review]

**Review by MichaelCurrin — APPROVED** [review]

**Comment by DirtyF** [comment]
@MichaelCurrin if you're interested in joining the Documentation team please join on https://teams.jekyllrb.com

**Review by DirtyF — APPROVED** [review]

**Comment by MichaelCurrin** [comment]
Thanks @DirtyF for the link. I hadn't seen that site before.

I've joined

**Comment by ashmaroli** [comment]
Welcome aboard @MichaelCurrin! Great to have you as part of the Team!

**Pull request merged**